ANSWERS.md

```diff
@@ -1,4 +1,14 @@
 - [ ] Why would you use class component over function components (removing hooks from the question)?
 - [ ] Name three lifecycle methods and their purposes.
 - [ ] What is the purpose of a custom hook?
-- [ ] Why is it important to test our apps?
\ No newline at end of file
+- [ ] Why is it important to test our apps?
+- [ ] Why would you use class component over function components (removing hooks from the question)?
+employer requests it... 
```

Comment:
Class components are useful for instantiating state when not using hooks in React.